iris-mpc-cpu/src/execution/hawk_main.rs

```diff
@@ -485,6 +498,13 @@ impl HawkActor {
     }
 }
 
+pub fn session_seeded_rng(base_seed: u64, store_id: StoreId, session_id: SessionId) -> ChaCha8Rng {
+    let mut hasher = DefaultHasher::new();
```

Comment:
There is no specification of what this is. Better use explicitly `siphasher::sip::SipHasher13`.